plugins/header_rewrite/operators.cc

```diff
@@ -1219,3 +1304,60 @@ OperatorRunPlugin::exec(const Resources &res) const
     _plugin->doRemap(res.txnp, res._rri);
   }
 }
+
+// OperatorSetBody
+void
+OperatorSetBodyFrom::initialize(Parser &p)
+{
+  Operator::initialize(p);
+  // we want the arg since body only takes one value
+  _value.set_value(p.get_arg());
+  require_resources(RSRC_SERVER_RESPONSE_HEADERS);
+  require_resources(RSRC_RESPONSE_STATUS);
+}
+
+void
+OperatorSetBodyFrom::initialize_hooks()
+{
+  add_allowed_hook(TS_HTTP_READ_RESPONSE_HDR_HOOK);
```

Comment:
I notice that your test scenario is using header_rewrite as a remap plugin.  Would this operator ever be useful when header_rewrite is used as a global plugin?

Reply:
Yes - this works with both remap and plugin configs. I have just added some tests to check this aswell.